### PR TITLE
Testing & CI

### PR DESCRIPTION
**Pull request by Trial-In-Error**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: lambdas/catalog-updater/__tests__/catalog-updater-test.js [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    test('should recognize custom domain name swagger files after fetching them from S3', async () => {
+        // custom domain name swagger files have their api id and stage name in their filename
+        // in the future, we should vastly prefer this over way of recognizing the file over the other forms
+
+        let fileBody = fs.readFileSync(path.join(__dirname, '/api-gateway-swagger.json')),
+            internalRepr
+
+        index.s3.getObject = jest.fn().mockReturnValueOnce(promiser({ Body: fileBody }))
+
+        internalRepr = await index.getSwaggerFile({Key: 'catalog/a1b2c3d4e5_customdomain.json'})
```
Comment (MynockSpit) [inline lambdas/catalog-updater/__tests__/catalog-updater-test.js]: Update to `stageName` or something else that indicates that this is in fact a stage and not the custom domain name.

File: lambdas/catalog-updater/__tests__/catalog-updater-test.js [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+    test('should recognize custom domain name OAS3 files after fetching them from S3', async () => {
+        // custom domain name OAS3 files have their api id and stage name in their filename
+        // in the future, we should vastly prefer this over way of recognizing the file over the other forms
+
+        let fileBody = fs.readFileSync(path.join(__dirname, '/api-gateway-oas3.json')),
+            internalRepr
+
+        index.s3.getObject = jest.fn().mockReturnValueOnce(promiser({ Body: fileBody }))
+
+        internalRepr = await index.getSwaggerFile({Key: 'catalog/a1b2c3d4e5_oascustomdomain.json'})
```
Comment (MynockSpit) [inline lambdas/catalog-updater/__tests__/catalog-updater-test.js]: Update to`stageName` or something else that indicates that this is in fact a stage and not the custom domain name. (Part 2)

File: lambdas/static-asset-uploader/__tests__/asset-uploader-test.js [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+            ResourceProperties: {
+                BucketName: 'bucketName'
+            },
+            RequestType: 'Delete'
+        },
+        context = {
+            test: 'context'
+        }
+
+        process.env.StaticBucketName = 'staticBucketName'
```
Comment (nitishdayal) [inline lambdas/static-asset-uploader/__tests__/asset-uploader-test.js]: Unset value from process pls

File: lambdas/static-asset-uploader/__tests__/asset-uploader-test.js [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+                ResourceProperties: {
+                    BucketName: 'bucketName'
+                },
+                RequestType: 'Create'
+            },
+            context = {
+                test: 'context'
+            }
+
+        process.env.StaticBucketName = 'staticBucketName'
```
Comment (nitishdayal) [inline lambdas/static-asset-uploader/__tests__/asset-uploader-test.js]: Same as above

**Pull request merged**